src/utils.rs

```diff
@@ -33,6 +33,20 @@ pub async fn create_response_text(
     create_embed_response(http, interaction, embed).await
 }
 
+pub async fn create_error_text(
+    http: &Arc<Http>,
+    interaction: &mut ApplicationCommandInteraction,
+    content: &str,
+) -> Result<(), ParrotError> {
+    let mut embed = CreateEmbed::default();
+    embed.description(content);
+    interaction
+        .create_followup_message(&http, |followup| followup.add_embed(embed))
+        .await
+        .map(|_| ())
```

Comment:
Do we need this mapping? Can we terminate with .await?

Reply:
It was required to be a drop in replacement for create_response_text, but I'll drop it to make this more generic for future follow up message usage